### PR TITLE
feat: add workflow recipe for translations

### PR DESCRIPTION
**Pull request by davidsoderberg**

### CODE REVIEW

File: recipes/workflows/translations.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: 'Translations'
+sidebarTitle: 'Translations'
+description: 'Translate notification content to different languages using i18n'
+---
+
+Translation allows you to seamlessly adapt your workflows to different languages and automatically apply them based on your users locales. It enhances engagement, personalizes the user experience, and allows you to expand your market reach.
```
Comment (rifont) [inline recipes/workflows/translations.mdx:7]: Some wording suggestions

```suggestion
Translations allow you to seamlessly adapt your notification workflows to different languages and automatically apply them based on your users locale. Translations enhance engagement, personalize the user experience, and enables you to expand your market reach.
```

File: recipes/workflows/translations.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+**Use case:** Facebook is a social media with users from all over the world. It allows the user to use Facebook in their language and expect to get emails from Facebook in their selected language.
```
Comment (rifont) [inline recipes/workflows/translations.mdx:9]:
```suggestion
**Use case:** Facebook is a social media platform with users from all over the world. Facebook allows the user to use Facebook in their language and expect to get emails from Facebook in their selected language.
```

File: recipes/workflows/translations.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+i18next.init({
+  fallbackLng: 'en',
+  resources: {
+    en: {
+      translation: en,
+    },
+    de: {
+      translation: de,
+    },
```
Comment (jainpawan21) [inline recipes/workflows/translations.mdx:27]: @davidsoderberg 

In the Novu v1 translation, we used format `en_US`, `de_DE` and recommend the same format for `subscriber.locale` field. Shall we use this format here?

Comment (davidsoderberg) [reply recipes/workflows/translations.mdx:27]: No as it is not needed really, I actually like to use other keys here to disconnect them from each other. Maybe it should be added in this recipe that they do not relate to each other what do you think?

Comment (jainpawan21) [reply recipes/workflows/translations.mdx:27]: As you are referencing `subscriber.locale` [here](https://github.com/novuhq/docs/pull/670/files#diff-2c229eda406d79bd407aee5aa8b92dc52fc091862d9d9bbd8cf1c8d9c37a2f4dR36). Wouldn't it break for users who are using V1 have locale already stored want to test v2.

I agree with you. We can add info about this in this recipe

File: recipes/workflows/translations.mdx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+title: 'Translations'
+sidebarTitle: 'Translations'
+description: 'Translate notification content to different languages using i18n'
+---
+
+Translation allows you to seamlessly adapt your workflows to different languages and automatically apply them based on your users locales. It enhances engagement, personalizes the user experience, and allows you to expand your market reach.
+
+**Use case:** Facebook is a social media with users from all over the world. It allows the user to use Facebook in their language and expect to get emails from Facebook in their selected language.
+
+In the example below uses [i18n](https://www.npmjs.com/package/i18n) in simple text email but if you use React to create your emails you can use for example [react-i18next](https://react.i18next.com/) and then just pass the variables needed for it. Before using i18n to translate, locale on the subscriber is used to set to the language that this email should be translated to.
```
Comment (rifont) [inline recipes/workflows/translations.mdx:11]:
```suggestion
The following example uses [i18n](https://www.npmjs.com/package/i18n) in simple text email, but if you use React to create your emails you can use [react-i18next](https://react.i18next.com/) and pass the variables needed for it. Before using i18n to translate, the subscriber's locale is used to set to the language that this email should be translated to.
```

**Review by rifont — APPROVED** [review]
Awesome!

File: recipes/workflows/translations.mdx (hunk 3 above)
Comment (davidsoderberg) [reply recipes/workflows/translations.mdx:27]: Yeah that would not work, but if we add a sentance about that this is not related to v1 I think it is okay, they can also edit the example before running it.

**Review by SokratisVidros — APPROVED** [review]

File: recipes/workflows/translations.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+description: 'Translate notification content to different languages using i18n'
+---
+
+Translations allow you to seamlessly adapt your notification workflows to different languages and automatically apply them based on your users locale. Translations enhance engagement, personalize the user experience, and enables you to expand your market reach.
+
+**Use case:** Facebook is a social media platform with users from all over the world. Facebook allows the user to use Facebook in their language and expect to get emails from Facebook in their selected language.
+
+The following example uses [i18n](https://www.npmjs.com/package/i18n) in a simple text email, but if you use React to create your emails you can use [react-i18next](https://react.i18next.com/) and pass the variables needed for it. Before using i18n to translate, the subscriber's locale is used to set to the language that this email should be translated to.
+
+**Note:** Translations in `@novu/framework` have no connection to translations in [v0.x](https://v0.x-docs.novu.co/content-creation-design/translations).
```
Comment (SokratisVidros) [inline recipes/workflows/translations.mdx:13]:
```suggestion
**Note:** Translations in `@novu/framework` are entirely separate from translations in [v0.x](https://v0.x-docs.novu.co/content-creation-design/translations).
```

**Review by SokratisVidros — APPROVED** [review]
❤️

**Review by rifont — APPROVED** [review]

**Pull request merged**